scss/custom-tagify.scss

```diff
@@ -0,0 +1,34 @@
+@import "../node_modules/@yaireo/tagify/src/tagify.scss";
+@import "variables"; 
+
+// Placeholder style for Tagify input
+.tagify__input::placeholder {
+  color: $dark;
+}
+
+// Hover state for Tagify tags
+.tagify__tag:hover {
+  background-color: $chocolate; // Use 'chocolate' for better contrast and theme consistency
+  border-color: lighten($chocolate, 10%); // Lighten the chocolate color for border
+}
+
+// Style for the remove button in tags
+.tagify__tag .tagify__tag__removeBtn {
+  background-color: $text-color-dark;
+}
+
+// Hover state for the remove button in tags
+.tagify__tag .tagify__tag__removeBtn:hover {
+  background-color: $chocolate; // Same 'chocolate' color for consistency
+}
+
+// Hover state for dropdown items
+.tagify__dropdown__item--active,
```

Comment:
this rule does not seems to work, but I don't know why.

Reply:
Hey thank you for the feedback. Just pushed changes to fix this.